tui/channels.sh

```diff
@@ -6,6 +6,9 @@ source "tui/locales/$LOCALE/channels.sh"
 active_channel="development"
 available_channels=(development)
 
+# adding back button
+available_channels+=($BACK_BUTTON)
```

Comment:
`$BACK_BUTTON` should not be in this array and I think it should not be in any array at all.